ci-operator/step-registry/cucushift/installer/check/vsphere/zone/rhcos-template/cucushift-installer-check-vsphere-zone-rhcos-template-commands.sh

```diff
@@ -14,22 +14,27 @@ source "${SHARED_DIR}/vsphere_context.sh"
 unset SSL_CERT_FILE
 unset GOVC_TLS_CA_CERTS
 INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
-
+dir=/tmp/installer
```

Comment:
Can you confirm the install log exists in the folder in this step, though `ipi-install-install` used the step, but the folder is only for that step, it is not shared with other steps.

Reply:
This variable can be removed now.